src/main/java/com/attica/athens/domain/agora/vote/api/AgoraVoteController.java

```diff
@@ -0,0 +1,51 @@
+package com.attica.athens.domain.agora.vote.api;
+
+import com.attica.athens.domain.agora.vote.application.AgoraVoteService;
+import com.attica.athens.domain.agora.vote.dto.request.AgoraVoteRequest;
+import com.attica.athens.domain.agora.vote.dto.response.AgoraVoteResponse;
+import com.attica.athens.domain.agora.vote.dto.response.AgoraVoteResultResponse;
+import com.attica.athens.domain.common.ApiResponse;
+import com.attica.athens.domain.common.ApiUtil;
+import com.attica.athens.global.auth.CustomUserDetails;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/v1/auth/agoras")
+public class AgoraVoteController {
+
+    private final AgoraVoteService agoraVoteService;
+
+    public AgoraVoteController(AgoraVoteService agoraVoteService) {
+        this.agoraVoteService = agoraVoteService;
+    }
+
+    @PatchMapping("/{agora-id}/vote")
+    public ResponseEntity<ApiResponse<?>> vote(
+            @AuthenticationPrincipal CustomUserDetails userDetails,
+            @PathVariable("agora-id") Long agoraId,
+            @RequestBody AgoraVoteRequest agoraVoteRequest
+    ) {
+
+        AgoraVoteResponse agoraVoteProsResponse = agoraVoteService.vote(userDetails.getUserId(), agoraVoteRequest,
+                agoraId);
+
+        return ResponseEntity.ok(ApiUtil.success(agoraVoteProsResponse));
+    }
+
+    @GetMapping("/{agora-id}/voteResult")
```

Comment:
p2: [REST API 가이드](https://covenant.tistory.com/241) 읽어보시면 좋을 것 같아요!
```suggestion
    @GetMapping("/{agoraId}/results")
```

Reply:
현재 API들이 대부분 path variable인데, 카멜 케이스가 맞지 않나요?